fetch_exceptions.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Recommend also adding the LICENSE file (https://www.mozilla.org/MPL/2.0/index.txt) /nitpick